src/WooCommerce.php

```diff
@@ -723,6 +723,11 @@ public static function woocommerce_checkout_process() {
     Server::addDebugMessage();
     if (!isset($response['statuscode']) || $response['statuscode'] !== 200) {
       $message = isset($response['userMessages']) ? implode('<br>', $response['userMessages']) : __('Error while saving the changes.');
+      // Add a note to explain what customers should do in case the mapping
+      // remains to fail.
+      $message .= vsprintf(' Sollte das Problem fortbestehen, <a href="%s" target="_blank">kontaktieren Sie bitte unseren Kundenservice</a>.', [
```

Comment:
We've been asked to replace the error message entirely (ignoring the actual error message returned by the API).